packages/editor/src/components/post-publish-panel/prepublish.js

```diff
@@ -50,6 +98,13 @@ function PostPublishPanelPrepublish( {
 				<strong>{ prePublishTitle }</strong>
 			</div>
 			<p>{ prePublishBodyText }</p>
+			<div className="components-site-card">
```

Comment:
We should turn this into a self contained component or components (site icon, site title)